pkgs/development/python-modules/json-stream/default.nix

```
@@ -25,6 +26,7 @@ buildPythonPackage rec {
 
```

```suggestion

  buildInputs = [
    iconv
  ];

```
starts to fix the darwin build failure